src/Routes.js

```diff
@@ -12,6 +13,7 @@ export default () => (
       <Route exact path="/login" component={LoginContainer} />
       <Route exact path="/signup" component={SignUpContainer} />
       <Route exact path="/dashboard" component={DashboardContainer} />
+      <Route exact path="/trips" component={TripView} />
```

Comment:
idk if I love this name since it's only showing data for a single trip... But it's also not a huge deal

Reply:
Very true. Would just doing `path="/trips"` be better?

Reply:
Or how about using `path="itinerary"`?

Reply:
I think itinerary works! Or even just trip singular... I do think it should be exact path though, otherwise you might get weird issues with the stuff you append?

Reply:
Yup! I'll make that change now. I think I'll go with `/trip` because it's more consistent with the terminology we've used to date.